sanowret-crystal.lic

```diff
@@ -14,6 +14,7 @@ class SanowretCrystal
     @invalid_rooms = []
     @worn_crystal = false
     args = parse_args(arg_definitions)
+    @adjective = get_settings.sanowret_adjective ? get_settings.sanowret_adjective : 'sanowret'
```

Comment:
❌ Since this is a new setting, it needs to be in `base.yaml` with a default value. If you make the default 'sanowret' you could avoid the conditional on this line.